### PR TITLE
Augument aggregate member names

### PR DESCRIPTION
**Pull request by merose**
Change aggregate member object names in OpenMCT to reflect the aggregate nesting. Add a new alias namespace, OpenMCT:omit, to cause a Yamcs parameter to be omitted from the object tree. Clean up iteration over parameter data by using iteration functions instead of for loops.

This resolves VIPERGDS-295.

### CODE REVIEW

**Review by akhenry — APPROVED** [review]
Looks fine to me. It's good to go once the other PRs are merged.

File: src/providers/object-provider.js
Comment (deeptailor) [inline src/providers/object-provider.js:89]: Since we expect parameters.parameters to be an Array, could we use Array.isArray(parameters.parameters), instead of just checking if its undefined.

Comment (merose) [reply src/providers/object-provider.js:89]: That would be slightly slicker. Since Andrew's already approved, I will leave refinements up to the OpenMCT team, I think. In practice, because Yamcs has a fixed output format based on a Protobuf definition, the "parameters" attribute will either be missing or an array.

**Pull request merged**
